### PR TITLE
sci-biology/qrna: add missing SLOT for hmmer

### PR DESCRIPTION
**Pull request by mmokrejs**

### CODE REVIEW

File: sci-biology/qrna/qrna-2.0.3c-r2.ebuild [hunk 1]
```diff
@@ -16,7 +16,7 @@ IUSE=""
 
 RDEPEND="
 	dev-lang/perl
-	sci-biology/hmmer"
+	sci-biology/hmmer:2"
```
Comment (a17r) [inline sci-biology/qrna/qrna-2.0.3c-r2.ebuild]: Thanks, this will require a revbump, so please bump this to -r3 and remove -r2 in a separate commit.

Comment (mmokrejs) [reply sci-biology/qrna/qrna-2.0.3c-r2.ebuild]: Done. BTW, what would be wrong with `git mv qrna-2.0.3c-r2.ebuild qrna-2.0.3c-r3.ebuild`?

**Comment by SoapGentoo** [comment]
Please squash your commits into one

File: sci-biology/qrna/qrna-2.0.3c-r2.ebuild (hunk 1 above)
Comment (a17r) [reply sci-biology/qrna/qrna-2.0.3c-r2.ebuild]: In general, easy ability to revert atomic changes. https://wiki.gentoo.org/wiki/Gentoo_GitHub#Separate_.28or_.22atomic.22.29_commits

**Comment by a17r** [comment]
Actually, is this really necessary? At least build of qrna seems to work fine with hmmer:0. Can you point at a bug?

**Comment by mmokrejs** [comment]
> Actually, is this really necessary? At least build of qrna seems to work fine with hmmer:0. Can you point at a bug?

Either repoman made me to do the changes or maybe I just wanted to mimick the same in other ebuilds. Sorry, I do not remember.

I tried to update the master (not this particular branch) but something went wrong again. I am sorry.

**Comment by mmokrejs** [comment]
Actually, is this really necessary? At least build of qrna seems to work fine with hmmer:0. Can you point at a bug?

hmmer 1 and 2 a 3 use different algorithms here and there but upstream refuses to rename binaries to distinguish they do different things. That is why hmmer:2 and hmmer:3 cannot coexist at the same time, unless omebody the hmmer itself na d3rd-party apps calling them.

I tried to update the master (not this particular branch) but something went wrong again. I am sorry.

**Pull request closed without merging**